### PR TITLE
feat: implement tabs

### PR DESCRIPTION
**Pull request by lehmi11**

### CODE REVIEW

**Comment by lehmi11** [comment]
I couldn't find a way to implement those animations from figma. Instead of wasting any more time, I implemented it without it

File: components/tabs/TabsItem.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <li className={clsx(
+      'cursor-pointer rounded-md py-[10px] px-[14px]',
+      isActive ? 'text-violet-600 bg-white ' : 'text-slate-600 hover:text-slate-800',
+
+    )}>
+      <button className={clsx(
+      )
+      }
+        onClick={onClick}>
+        <Label size='L' text={text} />
```
Comment (bladicito) [inline components/tabs/TabsItem.tsx]: I would not use a label inside a button. you can add the text  without a tag inside of the button. for accessibility also you can add the aria tags directly into the button tag. Up to you 👍🏽

File: components/tabs/TabsItem.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  text,
+  onClick,
+}: ITabsItemProps) => {
+  return (
+    <li className={clsx(
+      'cursor-pointer rounded-md py-[10px] px-[14px]',
+      isActive ? 'text-violet-600 bg-white ' : 'text-slate-600 hover:text-slate-800',
+
+    )}>
+      <button className={clsx(
```
Comment (bladicito) [inline components/tabs/TabsItem.tsx]: I think we could remove the clsx helper. right?

**Review by bladicito — APPROVED** [review]
Looks good to me. Thanks @lehmi11

File: components/tabs/TabsItem.tsx (hunk 1 above)
Comment (lehmi11) [reply components/tabs/TabsItem.tsx]: I removed <Label> from button element and used its text style instead

File: components/tabs/TabsItem.tsx (hunk 2 above)
Comment (lehmi11) [reply components/tabs/TabsItem.tsx]: yes. it is not needed.

**Pull request merged**